storage/tests/unit/test_blob.py

```diff
@@ -1164,7 +1165,8 @@ def _do_multipart_success(self, mock_get_boundary, size=None,
         stream = io.BytesIO(data)
         content_type = u'application/xml'
         response = blob._do_multipart_upload(
-            client, stream, content_type, size, num_retries, predefined_acl)
+            client, stream, content_type, size, num_retries, predefined_acl,
+            extra_headers)
```

Comment:
We should be exercising the headers in a new testcase, e.g. `test__do_multipart_upload_with_headers`, which passes actual headers to `_do_multipart_success`.  E.g.:

```python
    @mock.patch(u'google.resumable_media._upload.get_boundary',
                 return_value=b'==0==')
    def test__do_multipart_upload_with_headers(self, mock_get_boundary):
        self._do_multipart_success(
            mock_get_boundary,
            headers={'X-Goog-Testing': 'true'})
```